de.fu_berlin.inf.dpp.intellij/src/de/fu_berlin/inf/dpp/core/ui/util/CollaborationUtils.java

```diff
@@ -405,15 +420,17 @@ private static String getShareProjectDescription(
 
         }
 
-        HashMap<IProject, List<IResource>> resources = new HashMap<IProject, List<IResource>>();
+        HashMap<IReferencePoint, List<IResource>> resources = new HashMap<IReferencePoint, List<IResource>>();
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace the type specification in this constructor call with the diamond operator ("<>"). ([squid:S2293](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3AS2293))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=273&issues=AWaWrffYsSPV6ZXI_B9a&open=AWaWrffYsSPV6ZXI_B9a)

[ISSUE_KEY]: AWaWrffYsSPV6ZXI_B9a